bin/check-aggregate.rb

```diff
@@ -129,6 +129,13 @@ class CheckAggregate < Sensu::Plugin::Check::CLI
          description: 'Ignore severities, all non-ok will count for critical, critical_count, warning and warning_count option',
          boolean: true,
          default: false
+ 
+  option :switch_output,
+         short: '-R',
```

Comment:
Is there any real value to this one? `-R` is not exactly intuitive. I think it might be best to just forgo the short option.

Reply:
maybe if we go with `--verbose` a `-v` would make sense.